yarn.lock

```diff
@@ -3562,7 +3562,7 @@ enzyme-adapter-utils@^1.13.0:
     prop-types "^15.7.2"
     semver "^5.7.1"
 
-enzyme-shallow-equal@^1.0.1:
+enzyme-shallow-equal@^1.0.0, enzyme-shallow-equal@^1.0.1:
```

Comment:
You seem to have picked up some yarn updates. Maybe this has gotten out of sync on master.

Reply:
Weird, yeah.  OK, fixed that on a rebase...